File: src/targets/node/unirest/fixtures/jsonObj-null-value.js
const unirest = require('unirest');

const req = unirest('POST', 'http://mockbin.com/har');

req.headers({
  'content-type': 'application/json'
});

req.type('json');
req.send({
  foo: null
});

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

File: src/targets/python/python3/fixtures/jsonObj-null-value.py
import http.client

conn = http.client.HTTPConnection("mockbin.com")

payload = "{\"foo\":null}"

headers = { 'content-type': "application/json" }

conn.request("POST", "/har", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))